src/v2/components/Validators/Detail/index.jsx

```diff
@@ -108,9 +108,9 @@ const ValidatorsDetail = ({match}: {match: Match}) => {
       },
     },
     {
-      label: 'Missed blocks',
+      label: 'Amount of delegated sol',
```

Comment:
I think this needs a new label to make it clear that this is the validator's portion of everything staked in the network. The best I've come up with so far is "Amount of Total Staked SOL", although that seems really long and a bit awkward. Any ideas, @manuel-calavera or @sunnygleason ?

Reply:
@CriesofCarrots I think it is fine to keep since this label is being showed on validator detail page. So it is clear that all of these fields related to this particular validator.

Reply:
It is clear that the fields relate to this particular validator, but this field is a percentage of the *network total*. The network-total part is what I don't think is clear in the original label.

Reply:
Let's change this to 'Percent of Total Staked Sol'

Reply:
@sbhargava @CriesofCarrots may be just Total Staked Sol? It has sing of % after value. commission is also measured in % but we don't say Percent of Commission. Or we should?

Reply:
I get you on the % redundancy, but without "Percentage of", the word Total is ambiguous.
Here's another idea: Network Staked SOL ...?

Reply:
Ah, I see Chris wants to add Uptime to ValidatorDetail.
Given that, let's move this to inline with Staked SOL, so the field will appear:

Staked SOL(?) 0.0000XXX (0.XXX %)

Reply:
@CriesofCarrots nice! it is done

Reply:
🎊 Thank you!